webpack.config.babel.js

```diff
@@ -0,0 +1,47 @@
+import path from 'path';
+import DashboardPlugin from 'webpack-dashboard/plugin';
+import HtmlWebpackPlugin from 'html-webpack-plugin';
+
+
+export default {
+    entry: './src/js/index.js',
+    output: {
+        filename: 'bundle.js',
+        path: path.resolve(__dirname, 'dist/build')
+    },
+    resolve: {
+        alias: {
+            '~': path.resolve(__dirname, 'src/js')
+        }
+    },
+    module: {
+        rules: [
+            {
+                test: /\.js$/,
+                exclude: /(node_modules|bower_components)/,
+                use: [
+                    {
+                        loader: 'babel-loader',
+                        options: {
+                            presets: ['env', 'react']
+                        }
+                    },
+                    {
+                        loader: 'eslint-loader'
+                    }
+                ]
+            },
+        ]
+    },
+    devServer: {
+        contentBase: path.join(__dirname, 'src/js'),
+        compress: true,
+        port: 9000
+    },
+    plugins: [
+        new DashboardPlugin(),
```

Comment:
When you run npm start, it will load things into a dashboard looking thing, instead of just free flowing text.  Thought i'd give it a try cuz i saw someone else doing it -- looks pretty cool